openassessment/xblock/test/test_submission.py

```diff
@@ -98,6 +102,63 @@ def test_prompt_line_breaks(self, xblock):
         expected_prompt = u"<p><br />Line 1</p><p>Line 2</p><p>Line 3<br /></p>"
         self.assertIn(expected_prompt, resp)
 
+    @mock_s3
+    @override_settings(
+        AWS_ACCESS_KEY_ID='foobar',
+        AWS_SECRET_ACCESS_KEY='bizbaz',
+        FILE_UPLOAD_STORAGE_BUCKET_NAME="mybucket"
+    )
+    @scenario('data/file_upload_scenario.xml')
+    def test_upload_url(self, xblock):
+        xblock.xmodule_runtime = Mock(
```

Comment:
It would be nice if we can add doc strings for unit tests that are newly added at-least.